gulp.d/tasks/build.js

```diff
@@ -66,7 +66,7 @@ module.exports = (src, dest, preview) => () => {
       // NOTE concat already uses stat from newest combined file
       .pipe(concat('js/site.js')),
     vfs
-      .src('js/vendor/*([^.])?(.bundle).js', { ...opts, read: false })
+      .src(['js/vendor/*([^.])?(.bundle).js', '!js/vendor/kapa-widget.bundle.js'], { ...opts, read: false })
```

Comment:
Why did we need this change?

Did you try using the method we used for the other vendor scripts that don't get bundled? For example prism or algolia?

Reply:
if you remove `.bundle` from the script name, it shouldn't require this special treatment,